src/main/java/org/folio/search/utils/SearchUtils.java

```diff
@@ -18,6 +18,8 @@
   public static final String PLAIN_MULTILANG_PREFIX = "plain_";
   public static final String DOT = ".";
 
+  public static final int MAX_ELASTICSEARCH_QUERY_SIZE = 10_000;
```

Comment:
Is it a limit? If yes, I think we should confirm it with Magda. As far as I remember for inventory equivalent API we used 65 000 as a limit.

Reply:
yes, this is limit from elasticsearch: https://www.elastic.co/guide/en/elasticsearch/reference/7.12/paginate-search-results.html